### PR TITLE
Changed Personal and Company Address Labels

### PR DESCRIPTION
**Pull request by Luke9389**
### Fixed Issues

$ https://github.com/Expensify/App/issues/4631

### Tests / Web QA

1. Head over to http://localhost:8080/bank-account/new or http://staging.new.expensify.com/bank-account/new
2. Click `Connect Manually`
3. Enter `123123123` for both fields
4. Observe the label on the Company Address input.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [ ] Android

#### Web

Screen Shot 2021-08-16 at 11 33 21 AM

#### Android

cc @MitchExpensify

### CODE REVIEW

**Comment by Luke9389** [comment]
I'm assuming @shawnborton will want a little bit more space between the input and the text, but anything else Shawn?

File: src/pages/EnablePayments/AdditionalDetailsStep.js [hunk 1]
```diff
@@ -128,17 +128,20 @@ class AdditionalDetailsStep extends React.Component {
...
-                                    label={field.label}
-                                    onChangeText={val => this.setState({[field.fieldName]: val})}
-                                    value={this.state[field.fieldName]}
-                                    errorText={errorFields.includes(field.fieldName)
-                                        ? `${field.label} ${this.requiredText}`
-                                        : ''}
-                                    // eslint-disable-next-line react/jsx-props-no-spreading
-                                    {..._.omit(field, ['label', 'fieldName'])}
-                                />
+                                <Fragment>
```
Comment (Jag96) [inline src/pages/EnablePayments/AdditionalDetailsStep.js:131]: Looks like the linter caught this, but you can just use `<>` instead for this

**Review by Jag96 — COMMENTED** [review]
Looks good, just left the 1 comment for the issue caught by the linter. Will let design have a look before testing.

**Comment by shawnborton** [comment]
Can we use our hint text styles for the text that goes under the input? It should be at 13pt (in our fontSizeLabel var), in our colorMuted text color, and have a margin between the label and the input of 8px (or `mt2`). We might already have a style for it.

**Comment by MitchExpensify** [comment]
+1 to @shawnborton

**Comment by Luke9389** [comment]
Updated the code and the screenshot in the description. Lemme know what you think @shawnborton 
I wouldn't mind scooting the message to the right but it depends on whether that's consistent with our design philosophy.

**Comment by shawnborton** [comment]
Looks good! I like it aligned to the left, I think that is consistent with other places as well. The only thing I am curious about is perhaps we want to make the gap between input and hint just 4px, this way it looks more like the hint is tied to the input above it? Otherwise right now it kinda floats perfectly in between inputs and doesn't look quite visually connected to what's above. Thoughts on that?

**Comment by Luke9389** [comment]
Yea I think it looks better with 4px 👍
Screen Shot 2021-08-16 at 11 33 21 AM

**Comment by shawnborton** [comment]
Looks good to me, thanks Luke! All yours @Jag96 and @tylerkaraszewski

**Review by Jag96 — APPROVED** [review]
LGTM

**Comment by Jag96** [comment]
Going to CP this to staging to check it off the deploy checklist

**Pull request merged**